### DISCUSSION TITLE
W3C Compliant AnonCreds credentials in AFJ

### DISCUSSION THREAD

**Discussion by TimoGlastra** [openwallet-foundation/credo-ts#1671] [Architecture & Design]
Current working document with design (will keep this updated): 
https://hackmd.io/atAmNhg7QyOVUI_CMsW_-w?view

**Comment by TimoGlastra** [comment]
@cvarjao @swcurran this is our design document for the W3C representation of AnonCreds in AFJ. We're still making improvements to the document, especially the exchange protocol part (DIF PEX and Issue Attachment RFC), but it should give a good idea of how we want to approach this

**Comment by swcurran** [comment]
A few comments added. Looking forward to the discussion about the new Attachment type RFC.

**Comment by TimoGlastra** [comment]
RFC submission for credential attachment: https://github.com/hyperledger/aries-rfcs/pull/809

**Comment by TimoGlastra** [comment]
Done!
